aspnet/web-forms/overview/getting-started/hands-on-labs/using-page-inspector-in-visual-studio-2012.md

```diff
@@ -120,7 +120,7 @@ In this task, you will learn how to use the Page Inspector in the context of an
     *Toggle Inspection Mode button*
 6. In the Page Inspector browser, move the mouse pointer over the page elements. While you move the mouse pointer over any part of the rendered page, the element type is displayed and the corresponding source markup or code is highlighted in the Visual Studio editor.
 
-    ![Inspection mode in action](using-page-inspector-in-visual-studio-2012/_static/image5.png)
+    ![Screenshot showing the Page Inspector window and the Visual Studio editor with the element type displayed and the corresponding source markup is highlighted.](using-page-inspector-in-visual-studio-2012/_static/image5.png)
```

Comment:
...with the element type displayed and the corresponding source markup highlighted.